### ISSUE TITLE
Cannot generate test files.

### ISSUE THREAD

**Issue by mastrain** [f0uriest/keras2c#3]
I am using the repository as is in Master.  The test generation fails and does not create the C file.
I am using VS code in windows, using an anaconda enviroment to access tensorflow. This is the output:
```
All checks passed
Gathering Weights
Writing layer  input_1:0
Writing layer  dense_1
Writing layer  dense_2
Writing layer  lstm_1
Writing layer  dense_3
Traceback (most recent call last):
  File "c:/Users/migel/Repositorios/test_keras2c/keras2c/mytest.py", line 9, in <module>
    k2c(model,function_name,malloc=True,num_tests=1,verbose=True)
  File "c:\Users\migel\Repositorios\test_keras2c\keras2c\keras2c\keras2c_main.py", line 218, in k2c
    malloc_vars, stateful = model2c(
  File "c:\Users\migel\Repositorios\test_keras2c\keras2c\keras2c\keras2c_main.py", line 97, in model2c
    if not subprocess.run(['astyle', '--version']).returncode:
  File "C:\Users\migel\anaconda3\envs\keras_env\lib\subprocess.py", line 489, in run
    with Popen(*popenargs, **kwargs) as process:
  File "C:\Users\migel\anaconda3\envs\keras_env\lib\subprocess.py", line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\migel\anaconda3\envs\keras_env\lib\subprocess.py", line 1307, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] El sistema no puede encontrar el archivo especificado (The system cannot find the specified file)
```

The generated files are the following, they seem to be ok, but the test is missing.
![image](https://user-images.githubusercontent.com/12514135/98469095-f497a500-21dd-11eb-9a6e-4f68cfe9172f.png)

I think the problem might be how I had to set-up my enviroment. The keras2c library is not installed in my anaconda enviroment. Tried adding the keras2c folder to the PYTHONPATH of my enviroment, but it is still not working
Any idea?
Thank you.

**Comment by gazuara** [comment]
Hi there! I got exactly the same problem.
I am using anaconda as well, and my tf and keras versions are, respectively, 1.13.1 and 2.2.4
Any idea about what is going on??
Thank you so much!

**Comment by marcelodesouzaee** [comment]
>   File "c:\Users\migel\Repositorios\test_keras2c\keras2c\keras2c\keras2c_main.py", line 97, in model2c
>     if not subprocess.run(['astyle', '--version']).returncode:

"astyle" is missing on your system.

**Comment by f0uriest** [comment]
So sorry for the massive delay, I wasn't getting notifications for this repo for some reason. @marcelodesouzaee is correct that the issue is just that it can't find ``astyle`` which is used to clean up the generated code formatting. However, it shouldn't have thrown an error if it's not found, it should have just skipped the formatting. This should be fixed in #5 and v1.0.1, feel free to reopen if it's still an issue
